Portfolio/index.html

```diff
@@ -1,16 +1,67 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Portfolio</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
-    </div>
-    <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
-    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
-  </body>
-</html>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Portfolio</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="index.css" />
+  </head>
+
+  <body style="margin: 0; padding: 0">
+    <div top="0">
+      <a href="../index.html">Go Back</a>
+    </div>
+    <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
+    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
+  </body>
+</html>
+
+<!-- portfolio structure-->
+<html>
+    <body>
+        <li>About</li>
+        <li>Skills</li>
+        <li>Contact</li>
+        
+

```

Comment:
The 'h2' element should be placed within a 'div' or other appropriate container element.